DependencyInjection/CmfRoutingExtension.php

```diff
@@ -371,14 +384,32 @@ private function loadConditionalEnhancer(array $config, ContainerBuilder $contai
         $container->addDefinitions($definitions);
     }
 
-    private function buildConditionalEnhancerDefintion($className, $source, $target, $map, $methods, $priority)
+    private function buildConditionalEnhancerDefintion($className, $source, $target, $maps, $priority)
     {
+        var_dump($maps);throw new \Exception;
+        if (1 === count($maps) && !array_key_exists('methods', reset($maps))) {
```

Comment:
we should allow manually setting `any` as an method value instead of the old defintion too. Can be confusing else. 
So

- old configuration => stays old (no method) => can be merged in one group per configuration block
- new configuration with any in methods = old configuration => can go in the same block